modules/core/src/test/scala/gem/Arbitraries.scala

```diff
@@ -32,6 +32,9 @@ trait Arbitraries extends gem.config.Arbitraries  {
       )
     }
 
+  implicit val cogLocation: Cogen[Location] =
```

Comment:
`Cogen` is the dual of `Gen`. It's needed in order to generate "interesting" arbitrary functions. See [Erik's talk](https://www.youtube.com/watch?v=UlZGZh7hfbs) at about 36:00 for a really good explanation.